src/test/java/spoon/test/compilationunit/TestCompilationUnit.java

```diff
@@ -148,7 +155,7 @@ public void testAddDeclaredTypeInCU() throws IOException {
 		assertEquals(3, cu.getDeclaredTypes().size());
 
 		CtType typeBla = launcher.getFactory().Class().create("spoon.test.model.Bla");
-		cu.addDeclaredType(typeBla);
```

Comment:
could we stay backward compatible here and keep addDeclaredType?

Conceptually, we add types not references.